src/mixins/observable.mixin.js

```diff
@@ -89,33 +97,30 @@
       }
     }
     // one object with key/value pairs was passed
-    else if (arguments.length === 1 && typeof arguments[0] === 'object') {
+    else if (typeof eventName === 'object' && typeof handler === 'undefined') {
```

Comment:
is this better in anyway or you just like it more?

Reply:
`arguments.length === 1` is wrong because if you pass undefined as the second arg in will return false